packages/blockchain-wallet-v4-frontend/src/data/components/bchTransactions/sagas.js

```diff
@@ -49,6 +49,28 @@ export default () => {
             ? ''
             : payload.xpub || payload.address
           yield put(actions.core.data.bch.fetchTransactions(onlyShow, true))
+          break
+        case 'status':
+          const filter = payload => {
+            switch (payload) {
+              case 'sent':
+                return 1
+              case 'received':
+                return 2
+              case 'transferred':
+                return 3
+              default:
+                
```

Comment:
is this intentional that default do not return anything?

Reply:
Yep, wanted it to just return undefined if it wasn't the 3 options. Default return is always needed for switch statements, right? I added a break as the default case.